kong/tests/test_kong.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+import os
+import subprocess
+import logging
+import requests
+import time
+
+from datadog_checks.kong import Kong
+from datadog_checks.checks import AgentCheck
+from datadog_checks.utils.common import get_docker_hostname
+
+log = logging.getLogger('test_kong')
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+CHECK_NAME = 'kong'
+HOST = get_docker_hostname()
+PORT = 8001
+
+STATUS_URL = 'http://{0}:{1}/status/'.format(HOST, PORT)
+
+CONFIG_STUBS = [
+    {
+        'kong_status_url': STATUS_URL,
+        'tags': ['first_instance']
+    },
+    {
+        'kong_status_url': STATUS_URL,
+        'tags': ['second_instance']
+    }
+]
+
+BAD_CONFIG = {
+    'kong_status_url': 'http://localhost:1111/status/'
+}
+
+GAUGES = [
+    'kong.total_requests',
+    'kong.connections_active',
+    'kong.connections_waiting',
+    'kong.connections_reading',
+    'kong.connections_accepted',
+    'kong.connections_writing',
+    'kong.connections_handled',
+]
+
+DATABASES = [
+    'acls',
+    'keyauth_credentials',
+    'hmacauth_credentials',
+    'oauth2_credentials',
+    'consumers',
+    'nodes',
+    'response_ratelimiting_metrics',
+    'ratelimiting_metrics',
+    'oauth2_tokens',
+    'plugins',
+    'oauth2_authorization_codes',
+    'apis',
+    'jwt_secrets',
+    'basicauth_credentials',
+]
+
+
+def wait_for_cluster():
+    for _ in xrange(0, 100):
+        res = None
+        res_open = None
```

Comment:
What's `res_open` for?

Reply:
apparently nothing anymore gonna remvoe it